### DISCUSSION TITLE
Deleted Code

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#100] [Team Posts]
YAY! More practice! Our mentors, although extremely rare ;P, have made a mistake. While attempting to push code to git, we created a new workspace and unintentionally erased the previous version. Now that you all are experts, we can assign 3 of you to recreate it on Thursday.

**Comment by coverbeek** [comment]
^^^^^Danny wrote this ;P^^^^^^^

**Comment by coverbeek** [comment]
And by the "mentors", Danny really meant to say "Chris". My apologies, team. Luckily, I believe both Josh and Matt have versions of the code that are pretty close to what we had come up with.

(this really is Chris, this time)

**Comment by MikasaSucasa** [comment]
Since we deleted the code, our drive train was incorrectly re-written. Please read the DifferentialDrive.class to understand why. **Hint** they say in the class general comments that something is "negated"

**Comment by MikasaSucasa** [comment]
**ALSO**, arcadeDrive has two methods. If you look at the first method, you will understand why our motors were so slow at low input speeds during testing today
